Clients/GUI/Scripts/SignInner.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DTO;
+using HttpWebRequestWrapperLib;
+using System;
+
+namespace GUI.Scripts
+{
+    public class SignInner
+    {
+#pragma warning disable CS8632
```

Comment:
Seems nice xD

Reply:
I guess that when you use such stuff, it's a sign of some problems in method's architecture

Reply:
Fixed